grisera_api/activity_execution/activity_execution_service_graphdb.py

```diff
@@ -42,20 +47,19 @@ def save_activity_execution(self, activity_execution: ActivityExecutionIn):
                 type(self.activity_service.get_activity(activity_execution.activity_id)) is not NotFoundByIdModel:
             self.graph_api_service.create_relationships(start_node=activity_execution_id,
                                                         end_node=activity_execution.activity_id,
-                                                        name="hasActivity")
+                                                        name="Activity")
```

Comment:
why this name changed? every relationship starts with `has...`

Reply:
fixed